crates/telio-wg/src/wg.rs

```diff
@@ -817,6 +820,43 @@ impl State {
 
         Ok(success)
     }
+
+    /// This is like the normal '==' but ignores the time_since_last_rx field
+    fn is_same_interface(lhs: &Interface, rhs: &Interface) -> bool {
+        fn is_same_peer(lhs: &Peer, rhs: &Peer) -> bool {
```

Comment:
Better if `is_same_peer` is on the outside, instead of embedded in `is_same_interface`. Makes function look cleaner and easier to read.

Reply:
There is no other user/caller of this function - I'm not sure if moving it outside to the wider scope is a good idea. I can do it if you are strongly convinced that it will be better - let me know.

Reply:
I feel it would be a better idea to move it out.
1) Aesthetics
2) It isn't used by any other function now. It might be in the future.

What do you think ?

Reply:
1. That is pretty subjective 😄 
2. If in future someone wants to use it, they can easily extract it. Until that time it's I think easier to reason about the code when the scope of visibility is smaller.

Despite what I wrote above, this is in the end imo very minor issue so I will move that function to the outside :)

Reply:
I'd leave it as it is :smile: if it will be needed in future, it could be extracted, like @tomaszklak said.